cluster/azure/aks-gitops/outputs.tf

```diff
@@ -1,3 +1,11 @@
 output "kubeconfig_done" {
   value = "${module.aks.kubeconfig_done}"
 }
+
+output "aks_flux_kubediff_done" {
+  value = "${module.aks.kubeconfig_done}_${module.flux.flux_done}_${module.kubediff.kubediff_done}"
+}
+
+output "cluster_derived_resource_group" {
```

Comment:
No where are we using the cluster_derived_resource_group as far as I can find.  This can be removed.